R/zzz.R

```diff
@@ -2,6 +2,18 @@
 
 .onLoad <- function(...) {
 
+
+  # use the `cli` package to send a message on startup to use `enable_modules()`
+  msg <- c("")
+  packageStartupMessage(
+    # rlang::inform("This message will appear only once per session.", .frequency = "once", .frequency_id = "sortable"),
+    rlang::inform(
+      cli::cli_text("To use sortable with shiny modules, run {.run sortable::enable_modules()} to opt into the new standard."),
+    .frequency = "once",
+    .frequency_id = "sortable"
+    )
+  )
```

Comment:
I believe this constant messaging and inconsistent behavior is harder to understand than a big "BREAKING CHANGE, all your sortable DOM IDs have been updated. <previous_format> is now <new_format>. You only need to worry if you have custom css. Thank you for your patience as we have learned from our mistakes." in the NEWS.md

This would remove the startup message, remove the sortable_env, remove the vignette for inconsistent behavior. 

I'd rather lean towards "at version X, the behavior changed to something simple and expected".